src/components/ButtonGroup/ButtonWrapper.tsx

```diff
@@ -22,7 +25,7 @@ export const ButtonWrapper = ({
   tooltipContainerSelector,
   tooltipMoveBy,
   ...otherProps
-} : ButtonWrapperProps & ButtonProps) => {
+}: ButtonWrapperProps & ButtonProps) => {
```

Comment:
can you make `ButtonWrapperProps` be the exact type you need, instead of using `&` here?